behat-tests/features/acquia_lift_simple_ab.feature

```diff
@@ -1,146 +1,154 @@
+  Scenario: Create the simplest A/B campaign from start to finish.
+    # I login with the marketer role.
+    Given I am logged in as a user with the "access administration pages,access toolbar,administer visitor actions,manage personalized content" permission
+    And I am on the homepage
+    When I click "Acquia Lift" in the "menu" region
+    Then I should visibly see the link "Campaigns" in the "lift_tray" region
+    And I should not visibly see the link "Variation sets" in the "lift_tray" region
+    And I should not visibly see the link "Goals" in the "lift_tray" region
+    And I should not visibly see the link "Reports" in the "lift_tray" region
+    And I should not visibly see the link "Status" in the "lift_tray" region
+
+    # I bring up the "Create a campaign" interface.
+    When I hover over "Campaigns" in the "lift_tray" region
+    Then I should see the link "Add campaign" in the "lift_tray" region
+    Then I should see the link "All campaigns" in the "lift_tray" region
+    When I click "Add campaign" in the "lift_tray" region
+    Then I should see the modal with title "Create a campaign"
+    And I should see the link "A/B test" in the "modal_content" region
+    And I should see the link "Custom Lift campaign" in the "modal_content" region
+
+    # I create a simple A/B test campaign.
+    When I click "A/B test" in the "modal_content" region
+    Then I should see the modal with title "Create a campaign"
+    And I should see the link "Change type of test" in the "modal_content" region
+    When I click "Change type of test" in the "modal_content" region
+    Then I should see the modal with title "Create a campaign"
+    And I should see the link "A/B test" in the "modal_content" region
+    And I should see the link "Custom Lift campaign" in the "modal_content" region
+    When I click "A/B test" in the "modal_content" region
+    Then I should see the modal with title "Create a campaign"
+    And I should see the link "Change type of test" in the "modal_content" region
+    When I fill in "My test campaign" for "edit-agent-basic-info-title"
+    And I press the "Create campaign" button
+
+    # I verify my campaign is created.
+    Then I should see the message "Click the element you want to change in Variation #1" in the messagebox
+    When I should see the text "Campaign: My test campaign" in the "lift_tray_campaign_header" region
+    And I should visibly see the link "Variations" in the "lift_tray" region
+    And I should see "0" for the "variation" count
+    And I should visibly see the link "Goals" in the "lift_tray" region
+    And I should see "0" for the "goal" count
+    And menu item "Reports" should be "inactive"
+    And menu item "Start campaign" should be "inactive"
+    And I should not see the modal
+    And the variation edit mode is "active"
+
+    # I create a new variation.
+    When I click "#site-name a span" element in the "page_content" region
```

Comment:
Yes at least for consistency, I think using article for all feature tests are better. I can switch if you feel it is needed.

On the other hand ... 

I was always hoping that we can separate the dev environment from behat test environment. i.e. test insulation in full. I have actually tried a bit, and it can be as simply done as during Behat run time:
# Create or clone over the full DB to DB_behat
# Add another drupal connection for the DB_behat DB and switch to it, 4) tear down and repeat by @beforescenario and @insulated. I got quite far actually, but didn't want to go pass my two-day time box for this story.

If I had to choose, I'd spend time figuring out this insulation instead of using means like article to dodge problems.